test/model/saved_snippet.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:checks/checks.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:zulip/api/model/events.dart';
+import 'package:zulip/api/model/model.dart';
+
+import '../api/model/model_checks.dart';
+import '../example_data.dart' as eg;
+import 'store_checks.dart';
+
+void main() {
+  test('handleSavedSnippetsEvent', () async {
+    final store = eg.store(initialSnapshot: eg.initialSnapshot(
+      savedSnippets: [eg.savedSnippet(id: 101)]));
+    check(store).savedSnippets.values.single.id.equals(101);
+
+    await store.handleEvent(SavedSnippetsAddEvent(id: 1,savedSnippet:
```

Comment:
nit: space after comma; actually maybe put `savedSnippet:` on the next line, with `eg.savedSnippet(`?